external-builds/pytorch/skip_tests/generic.py

```diff
@@ -13,6 +13,10 @@
         # "binary_ufuncs": [ "test_cuda_tensor_pow_scalar_tensor_cuda" ]
         # ----------------
         "cuda": [
+            # Always skip as this test is very basic and needs manual addition in the torch/test_cuda.py
+            # for new architectures to be run - and it just checks if hipblaslt is the preferred library
+            # for AMD Instinct GPUs.
+            "test_preferred_blas_library_settings",
```

Comment:
nit: sort skipped tests alphabetically (move to the bottom of this list)?

I also have this disabled on my latest windows test bringup branch: https://github.com/ROCm/TheRock/blob/cc9f0f30544ad84bbccd6f3b001ebadc670398f0/external-builds/pytorch/skip_tests/generic.py#L45-L49